client/devicemanager/manager.go

```diff
@@ -161,17 +158,12 @@ func (m *manager) Run() {
 		})
 	}
 
-	// XXX we should eventually remove this and have it be done in the client
```

Comment:
I am very happy with this path. I didn't expect this comment to go away for quite a while 😄